src/generic/listctrl.cpp

```diff
@@ -4480,9 +4478,11 @@ void wxListMainWindow::SortItems( wxListCtrlCompare fn, wxIntPtr data )
     HighlightAll(false);
     ResetCurrent();
 
-    list_ctrl_compare_func_2 = fn;
-    list_ctrl_compare_data = data;
-    m_lines.Sort( list_ctrl_compare_func_1 );
+    wxListLineComparator comparator;
```

Comment:
IMHO it would make sense to give this struct a ctor taking `fn` and `data`.

Reply:
This is like struct wxInternalDataSort in msw/listctrl.cpp.
 I'd like to keep it as PlainOldData type to avoid any issues with templates.